recipes/_install-windows.rb

```diff
@@ -35,6 +35,8 @@
 # Install the package
 windows_package 'Datadog Agent' do
   source temp_file
-  options %(APIKEY="#{node['datadog']['api_key']}" HOSTNAME="#{node['hostname']}" TAGS="#{node['tags'].join(',')}")
```

Comment:
This really isn't needed as its entirely managed by the template unless I am missing something?